config/alpha.json

```
@@ -12,6 +12,7 @@
   "jobManagerUrlRoot": "https://job-manager.dsde-alpha.broadinstitute.org/jobs",
   "leoUrlRoot": "https://leonardo.dsde-alpha.broadinstitute.org",
   "marthaUrlRoot": "https://us-central1-broad-dsde-alpha.cloudfunctions.net",
+  "metadataEndpoint": "martha_v2",
```

Not loving this name collision

If we insist on having metadata in the name (perhaps because we plan to have non-metadata Martha endpoints in the future?) we can do
```suggestion
  "marthaMetadataEndpoint": "martha_v2",
```

---

It has "metadata" in the name because Terra UI also calls the `getSignedUrlV1` Martha endpoint.